src/messages/serenity.proto

```diff
@@ -6,6 +6,14 @@ option java_package = "org.apache.mesos";
 option java_outer_classname = "Protos";
 
 
+message OversubscriptionReCtrlEventEnvelope {
```

Comment:
To separate RE and QoS messages. To resolve this we can easily add "string" classifier as topic.